prisma/seed.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { PrismaClient } from '@prisma/client';
+
+const facilities = require('./fixtures/facilities.json');
+const services = require('./fixtures/services.json');
+const treatments = require('./fixtures/treatments.json');
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const facilitiesPromise = facilities.map((facility) =>
```

Comment:
better to use `createMany` instead of `upsert`. It is supposed to execute only once when the database is new. So, no need to upsert.